DevCycle.SDK.Server.Local/DevCycle.SDK.Server.Local.csproj

```
@@ -29,7 +29,7 @@
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <LangVersion>latest</LangVersion>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

This is still targeting multiple - you want 
```suggestion
    <TargetFramework>netstandard2.0</TargetFramework>
```